Makefile

```diff
@@ -5,6 +5,7 @@ GITSHA:=$(shell git rev-parse HEAD)
 # Get the current local branch name from git (if we can, this may be blank)
 GITBRANCH:=$(shell git symbolic-ref --short HEAD 2>/dev/null)
 GOFMT_FILES?=$$(find . -not -path "./vendor/*" -name "*.go")
+BAD_FILES=$(shell echo $(GOFMT_FILES) | xargs gofmt -s -l)
```

Comment:
The 'echo' alone can still run off the end of the Command line length restriction.